ngraph/src/ngraph/op/ctc_loss.hpp

```diff
@@ -46,6 +46,14 @@ namespace ngraph
                 /// potential alignment
                 /// \param unique                         Flag to find unique elements in a target
                 /// before matching with alignment
+                CTCLoss(const Output<Node>& logits,
+                        const Output<Node>& logit_length,
+                        const Output<Node>& labels,
+                        const Output<Node>& label_length,
+                        const bool preprocess_collapse_repeated = false,
+                        const bool ctc_merge_repeated = true,
+                        const bool unique = false);
+
```

Comment:
Comment for line 75. AFAIK, we wanted to avoid statements like `using v4::CTCLoss;`. So we must explicitly specify the operation version in the code or use operation from specific opset.

@GlebKazantaev , @ilyachur , please, comment on this.

Reply:
Agree! Please remove `using v4::CTCLoss;`

Reply:
Fixed